uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -847,6 +847,76 @@ public function testFindOverlappingLeaveRequestsForOneOverlappingLeaveRequest()
     $this->assertEquals($leaveRequest1->id, $overlappingRequests[0]->id);
   }
 
+  public function testManagerCanCancelOrRejectUserLeaveRequestEvenIfBalanceChangeIsGreaterThanPeriodEntitlementBalanceChangeWhenAllowOveruseFalse() {
+    $manager = ContactFabricator::fabricate();
+    $staff = ContactFabricator::fabricate();
+    $entitlementFromDate = CRM_Utils_Date::processDate('2016-01-01');
+    $entitlementToDate = CRM_Utils_Date::processDate('2016-12-31');
+    $requestFromDate = new DateTime('2016-06-14');
+    $requestToDate = new DateTime('2016-06-14');
+    $requestFromDateType = $this->leaveRequestDayTypes['all_day']['value'];
+    $requestToDateType = $this->leaveRequestDayTypes['all_day']['value'];
+
+    $this->registerCurrentLoggedInContactInSession($manager['id']);
+    $this->setContactAsLeaveApproverOf($manager, $staff, null, null, true, 'has Leaves Approved By');
```

Comment:
Just this should be enough `$this->setContactAsLeaveApproverOf($manager, $staff);` since we only need to ensure contact has leave approver relationship with manager.

Reply:
Done.